src/messages/Image/lightbox/LightboxHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import { KeyboardEvent, useRef } from "react";
+import { useMessangerImageContext } from "../hooks";
+import classes from "./Lightbox.module.css";
+import { CloseIcon, DownloadIcon } from "src/assets/svg";
+
+const LightboxHeader = () => {
+	const { url, altText, onClose } = useMessangerImageContext();
+
+	const firstButton = useRef<HTMLButtonElement>(null);
+
+	const handleDownload = () => {
+		window.open(url, "_blank");
```

Comment:
Yes, it was the old behavior. When I built the Lightbox I remember I asked if we want the real download, and it was decided to keep like that. It can be handled anyway in a follow-up, if needed.